TombEngine/Game/effects/tomb4fx.cpp

```diff
@@ -1155,50 +1156,60 @@ void SomeSparkEffect(int x, int y, int z, int count)
 	}
 }
 
-void TriggerUnderwaterExplosion(ItemInfo* item, int flag)
+void TriggerUnderwaterExplosion(ItemInfo* item, bool flag)
```

Comment:
If `flag` argument meaning is clarified, please rename `flag` to something more meaningful. It's a remnant of decompilation.